data/random-teams.ts

```diff
@@ -906,7 +906,13 @@ export class RandomTeams {
 			return {cull: hasMove['encore'] || hasMove['nastyplot'] || hasMove['swordsdance']};
 		case 'thunderwave': case 'voltswitch':
 			const cullInDoubles = hasMove['electroweb'] || hasMove['nuzzle'];
-			return {cull: counter.setupType || counter.speedsetup || hasMove['raindance'] || (isDoubles && cullInDoubles)};
+			return {cull:
+				counter.setupType ||
+				counter.speedsetup ||
+				hasMove['shiftgear'] ||
+				hasMove['raindance'] ||
+				(isDoubles && cullInDoubles),
+			};
```

Comment:
```suggestion
			return {
				cull: counter.setupType ||
					counter.speedsetup ||
					hasMove['shiftgear'] ||
					hasMove['raindance'] ||
					(isDoubles && cullInDoubles),
			};
```

Very minor, but we prefer this style.

Reply:
oh! i was just matching some of the other parts of the file, but sure